### ISSUE TITLE
Is it possible to have multiple Fi bases and collars?

### ISSUE THREAD

**Issue by tkuennen** [sbabcock23/hass-tryfi#27]
When trying to set this up it only picked up one of our three dog collars/bases. Is it possible to add more than one?

**Comment by sbabcock23** [comment]
Hi @tkuennen,

Can you tell me if its failing to pick up the bases, the collars or both?

**Comment by tkuennen** [comment]
Hi @sbabcock23,

My apologies for the delayed response. It's both the bases and the collars. Another odd bit is that sometimes when I restart Home Assistant it will sometimes disconnect from one and connect to the next. Almost seems to be doing round-robbin between them. We have three dogs with Fi and I'd love to be able to check in on all three with HASS.

**Comment by causefx** [comment]
Is this a limitation of this component or TryFi?

**Comment by Chris-Jacobs** [comment]
I just set this up with two dogs/collars/bases and I'm seeing the same behavior as @tkuennen

**Comment by tkuennen** [comment]
I believe this to be a limitation with the component because while troubleshooting it would loop through our three collars however only one would be available at a time leaving the other two in an unknown state. I'm about to repave HASS and will do a bit of debugging on this. Will report back in Jan with what I find.

**Comment by causefx** [comment]
thanks @tkuennen

**Comment by ndragon798** [comment]
I got this to work with two collars 
![image](https://user-images.githubusercontent.com/10368650/146652399-a37123e5-c5ff-4289-bc3b-97895d989678.png)
I added another entry of tryfi to my config and configured it with my girlfriends email on the second instance. So it was querying my email and my girlfriends at the same time. You could probably just create a second email account and then add it as an owner.

**Comment by tkuennen** [comment]
@ndragon798 nice find. This worked for me as well.

**Comment by russalex** [comment]
Any chance on a quick mult-dog (collar & base) for the cheap seats?

I added a 2nd collar / dog to my TryFi account and now having some strange issues. At this point I'm seeing:

- light.dog1_collar_light
- select.dog1_lost_mode
- select.dog1_lost_mode
- light.dog2_collar_light
- select.dog2_lost_mode

I have confirmed I have the latest update in HACS.

Any suggestions?

**Comment by sbabcock23** [comment]
Hi @tkuennen and @russalex ,

Would you be able to enable debug on HA and share the tryfi specific log entries? There should be api calls with the data in it and errors potentially. Please sanitize any data as required before sharing. 

Thanks!

**Comment by sbabcock23** [comment]
I believe this has resolved. Please let me know if it's still an issue.

**Comment by russalex** [comment]
Doh, I didn't see the request on May 1st. Sorry

The good news, all appears to be working now. The two dogs are showing up correctly.  Thanks!

**Comment by ExodusOTH** [comment]
I hate to necro an old issue, but I'm experiencing this bug again. I've set my logging mode for the TryFi integration to debug, and it doesn't appear to even be aware that anything is incorrect; perhaps it's an issue with PyTryFi? The below logs are from a fresh restart and clean installation of the TryFi integration.

For context around my use case, myself and my brother both have TryFi accounts, he has one for his dog Scout, and I have one for my dog Nala. We have set eachother as a "Dog Owner" of eachother's dogs as well, so I have visibility into Scout's collar and he has visibility/control into Nala's collar. Despite logging in with my own credentials (not my brothers), it only seems to be aware of Scouts existence.

Any thoughts @sbabcock23?

[home-assistant.log](https://github.com/sbabcock23/hass-tryfi/files/9330572/home-assistant.log)

(Edited to just include the logs as a file)
